tests/kernel/install_kotd.pm

```diff
@@ -102,14 +103,25 @@ END
 
 sub run {
     my $self = shift;
+    my $default_version;
+    my $baseurl;
+
     $self->wait_boot;
 
     $self->select_serial_terminal;
 
-    my $url = get_required_var("INSTALL_KOTD");
+    if (is_sle) {
+        $default_version = "SLE" . get_required_var("VERSION");
```

Comment:
maybe you can use generate_version()  from repo_tools lib.

    $default_version = generate_version();

You would get something like SLE_12_SPX, SLE_15_SPX, etc

Reply:
But I need `SLE12-SP3` :(. Maybe there could be a parameter for `generate_version()` to keep `-` for SLES as well. But I'd like to do it as a separate approach.

Reply:
ok, then you can't use that function.

Reply:
> ok, then you can't use that function.

I could patch `generate_version`, but I don't see much benefits from it.